homeassistant/components/solaredge/const.py

```diff
@@ -149,35 +149,50 @@
         json_key="Purchased",
         name="Imported Power",
         entity_registry_enabled_default=False,
-        icon="mdi:flash",
+        icon="mdi:home-import-outline",
+        state_class=STATE_CLASS_MEASUREMENT,
+        unit_of_measurement=ENERGY_WATT_HOUR,
+        device_class=DEVICE_CLASS_ENERGY,
     ),
     SolarEdgeSensorEntityDescription(
         key="production_power",
         json_key="Production",
         name="Production Power",
         entity_registry_enabled_default=False,
-        icon="mdi:flash",
+        icon="mdi:solar-power",
+        state_class=STATE_CLASS_MEASUREMENT,
+        unit_of_measurement=ENERGY_WATT_HOUR,
```

Comment:
Isn't the unit provided by the API in this case?

Reply:
ENERGY_WATT_HOUR is imported by homeassistant.const. I need to check Solaredge API if unit is present.